hub/powertoys/newplus.md

```diff
@@ -0,0 +1,43 @@
+---
+title: PowerToys New+ for Windows
+description: A tool that enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
+ms.date: 07/13/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, New+, New, NewPlus, Win]
+---
+
+# New+
+
+**PowerToys New+** enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
```

Comment:
I try to avoid saying something is "easy" or "simple" in the docs in case somebody has difficulty getting something configured.
```suggestion
**PowerToys New+** enables you to create files and folders from a personalized set of templates, directly from the File Explorer context menu.
```